config/settings.yml

```
@@ -0,0 +1,4 @@
+length:
+  min_8: 8
```

```suggestion
  digit_8: 8
```

ri cho khả năng tái sử dụng cao hơn nhé